android/app/src/main/AndroidManifest.xml

```diff
@@ -54,6 +68,15 @@
             </intent-filter>
 
         </activity>
+        <provider
```

Comment:
Can you check if the app works without adding it? Wondering why it was working for set-as option.